cloud: add import snapshot files.md to toc (#21791)
pingcap/docs: tidb-cloud/import-snapshot-files-serverless.md

---
title: Import Snapshot Files into {{{ .starter }}} or Essential
summary: Learn how to import Amazon Aurora or RDS for MySQL snapshot files into {{{ .starter }}} or Essential.
---

# Import Snapshot Files into {{{ .starter }}} or Essential

> You can import snapshot files from Amazon Aurora or RDS for MySQL into {{{ .starter }}} or Essential. These snapshots are exported as Parquet files. To ensure a successful import, your data files must follow specific naming conventions. For example, all source data files must have a `.parquet` suffix and be located in a folder named `{db_name}.{table_name}/`. For complete details, see [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md).

Author: ```suggestion
You can import snapshot files from Amazon Aurora or RDS for MySQL into {{{ .starter }}} or Essential. You need to export these snapshot files from Amazon Aurora or RDS for MySQL as Parquet files first. To ensure a successful import, your data files must follow specific naming conventions. For example, each source data file must have a `.parquet` suffix and be located in a folder named `{db_name}.{table_name}/`. For complete details, see [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md).
```

> 

Reviewer: “These snapshots are exported as Parquet files." What does this mean? Users need to export these snapshot files from Amazon Aurora or RDS for MySQL as Parquet files first?

Reviewer 2: Yes, the snapshot format is Parquet by default.
